app/policies/recipe_policy.rb

```diff
@@ -0,0 +1,53 @@
+class RecipePolicy < ApplicationPolicy
+  attr_reader :user, :recipe
+
+  def initialize(user, recipe)
+    @user = user
+    @recipe = recipe
+  end
+
+  def show
```

Comment:
Got the wrong name, which means the method in `ApplicationPolicy ` is being used instead
```suggestion
  def show?
```

Reply:
This is one of those that confuse me lol. When is should I use the ? or not. I read it in the docs but still have trouble deciding.

Reply:
Common practice is to name methods that return true/false with a ?

Here specifically, I think pundit is inferring that if you are calling authorize from an action named foo, it should use the foo? method in the policy class